### DISCUSSION TITLE
Using SuperProductivity + Obsidian in research

### DISCUSSION THREAD

**Discussion by IanQS** [super-productivity/super-productivity#2766] [Q&A]
Hey all!

I was wondering if anyone uses SP for unstructured research tasks? Or is it meant more for daily tasks and TODOs?

I'm hoping to "integrate" it with Obsidian to create a supercharged research tool, but I'm wondering if anyone has any thoughts on:

- the efficacy of it
- the workflow

Thanks for reading!

**Comment by johannesjo** [comment]
That's a very interesting question! Personally I use other tools (mostly just an empty markdown doc or sometimes mindmaps) to do exploratory work, while I still do the time tracking in SP.

**Comment by Jagdfalke** [comment]
Personally I use both Obsidian and Super Productivity. Obsidian is my main tool for anything related to text, be it daily notes or meeting notes, documentation, research, you name it. 

Super Productivity is my main tool for organizing my work in general. Anything that is a task and requires an action from myself goes into Super Productivity. I use it to plan ahead, handle my workload and track my time.

Both workflows hardly overlap. The only overlap occurs when I regularly paste text from Obsidian into Super Productivity as a description for a task. But this is just a one time action to have everything at hand as soon as I start working on a task. The text doesnt evolve in Super Productivity, this is what Obsidian is there for.

**Reply by IanQS** [reply]
A year in (maybe more) is this still your workflow? Have you tweaked it since? I've been task tracking in Obsidian for daily and weekly tasks and it's alright so far, but I'm wondering if it could be better

**Comment by bepolymathe** [comment]
Could @johannesjo's announcement of a [plugin system development](https://github.com/johannesjo/super-productivity/discussions/4521) make it possible to consider such an integration? That would be ideal!

**Comment by JimBreton** [comment]
I find that (for me at least) research notes, thoughts, tasks, and the like tend to be strongly related to each other and context is very helpful to have close at hand when spinning out tasks.

For example while researching something and trying to figure it out, I may take notes on that topic; and this leads to questions and additional threads to tug on; those become additional research tasks that I may not take on at that very moment.  And _planned_ tasks can also benefit from that project's context (notes).

SP seems to be a great tool for task management, and to some extent managing project tasks as well, but it would be difficult for it to also solve the contextual notes problem without the ability to paste arbitrary clipboard content -- example screenshots (#2926).

I guess what I am saying is -- a good PKM is great, and a good task manager is great -- but having them both in a single tool is amazing.  It doesn't need to be a super complex tool either, just having some improved 'rich editing' capabilities would make leaps and bounds towards that ideal.

Currently Logseq is filling that role for me and works great!  There are very few tools that can solve both problems.  (Logseq, Obsidian, Notion, even OneNote to some extent.)  But the Logseq UX is abysmal compared to SP.

Anyway fantastic project here, I will be keeping an eye on this as well as recommending it to folks who are looking for a good task manager and not necessarily trying to solve both problems in a single tool. :) Thank you!

[a comment by an automated account is not shown]
